src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/hila/StreamBatch.java

```diff
@@ -0,0 +1,51 @@
+package de.zalando.aruha.nakadi.webservice.hila;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import de.zalando.aruha.nakadi.domain.Cursor;
+
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.Collections.unmodifiableList;
+
+@Immutable
+public class StreamBatch {
+
+    private Cursor cursor;
+    private List<Map> events;
```

Comment:
1) Map is basically <String, Object> but I don't see much sense to specify that as the key itself can be String or List or Map. So defining a map as <String, Object> doesn't really help.
2) Added finals, good point.